setup.py

```diff
@@ -29,14 +29,15 @@
       include_package_data=True,
       package_data={
             '': [
-                  '*.information', '*.qrc', '*.sql', '*.xml'
+                  '*.information', '*.qrc', '*.sql', '*.xml', "*.css", "*.png"
             ] + extra_package_data,
             'cc3d.player5': [
                   'compucell3d.pyw',
                   'Configuration/*',
                   'Configuration_settings/*',
                   'Configuration_settings/osx/*',
                   'icons/*',
+                  'styles/themes/*',
```

Comment:
Why do certain directories need to be included while others are not listed? (e.g. the `styles/` parent dir)